### PR TITLE
homework1

### PR DESCRIPTION
**Pull request by nazlukyanovich**
1. Mutants
2. Chess knights

### CODE REVIEW

File: koni.cpp [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	cin >> x1 >> y1;
+	cout << "Enter the coordinates of the second knight: " << endl;
+	cin >> x2 >> y2;
+	
+	if(abs(x2 - x1) == 2 && abs(y2 - y1) == 1 || abs(x2 - x1) == 1 && abs(y2 - y1) == 2){
+		cout << "Knight beats another one";
+	} else {
+		cout << "Knight dont touch another one";
+	}
+}
```
Comment (AlexanderRusak) [inline koni.cpp]: A user can put a figure on the other
A user can input negative coordinates its incorrect

File: mutanti.cpp [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#include <iostream>
+using namespace std;
+
+int main(){
+	int mutants, p, n, days, breeding;
+	days = 0;
```
Comment (AlexanderRusak) [inline mutanti.cpp]: All variables which you use in mathematical calulation must be in double type

File: mutanti.cpp [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+	
+	cin >> mutants;
+	cout << "The city was attacked by " << mutants << " mutants!!!" << endl;
+	cin >> p;
+	cout << "They multiply by " << p << "% every night" << endl;
+	cin >> n;
+	cout <<"If utilities kill " << n << " mutants every day then ";
```
Comment (AlexanderRusak) [inline mutanti.cpp]: Unuser friendly program

**Review by AlexanderRusak — COMMENTED** [review]
Correct mistakes

**Comment by nazlukyanovich** [comment]
fixed, thanks

File: mutanti.cpp [hunk 4]
```diff
@@ -17,7 +23,7 @@ int main(){
 		mutants = mutants + breeding - n;
 		days = i;
 		
-		if (breeding > n){
+		if (breeding >= n){
 			cout << "YOU DIED" << endl;
 			break;
 		}
```
Comment (AlexanderRusak) [inline mutanti.cpp]: Relook your logic in app

**Review by AlexanderRusak — APPROVED** [review]

**Pull request merged**